README.md

```diff
@@ -1 +1,132 @@
+section](https://buildroot.org/downloads/manual/manual.html#requirement).
+
+For Debian/Ubuntu distributions, the necessary packages can be installed
+with the following command:
+
+```bash
+$ sudo apt install debianutils sed make binutils build-essential gcc g++ bash patch gzip bzip2 perl tar cpio unzip rsync file bc git
+```
+
+Optional dependencies are available for enabling addiotnal Buildroot
+features such as interface configuration, legal information or documentation.
+For detailed information refer
+[corresponding manual section](https://buildroot.org/downloads/manual/manual.html#requirement-optional).
+
+### Steps to Build the Image
+
+Clone the Buildroot repository to local machine.
+
+```bash
+$ git clone -b 2024.05.1 https://github.com/bootlin/buildroot.git
+```
+
+Clone the `BR2_EXTERNAL` tree:
+
+```bash
+$ git clone -b 10.00.07 https://github.com/TexasInstruments/buildroot-external-TI.git
+```
+
+The directories for `buildroot` directory and `buildroot-external-ti`
+are now located side by side.
+
+### Configure and build
+
+Go to the Buildroot directory:
+
+```bash
+$ cd buildroot/
+```
+Configure Buildroot to use the external tree by setting the BR2_EXTERNAL
+environment variable. This tells Buildroot to include the configurations
+and packages from the external tree during the build process.
+To configure the system for build, choose any of the *defconfigs* 
+provided in this `BR2_EXTERNAL` tree. For example:
+
+```bash
+buildroot/ $ make BR2_EXTERNAL=../buildroot-external-TI ti_release_am62x_sk_defconfig
+```
+
+For further customize the Buildroot configuration, run `make menuconfig`.
+This step is optional but useful if you need to make specific adjustments.
+
+Build the Image:
+
+```bash
+buildroot/ $ make
+```
+
+This process compiles the necessary components and creates the root
+filesystem, kernel, and bootloader. The build process can take some
+time, depending on your system's resources and the complexity of
+the configuration.
+
+## Flashing and booting the image
+
+The Buildroot configurations generate a compressed ready-to-use SD card
+image, available as `output/images/sdcard.img`.
+
+Flash this image on a SD card:
+
+```bash
+buildroot/ $ sudo dd if=output/images/sdcard.img of=/dev/sdX
+```
+
+(Note: Replace /dev/sdX with the appropriate device identifier for your SD card.
+)
+
+Tools like balena-ethcer can also be used to flash this image.
+
+Insert the SD Card in SD Card slot and power on the board (ex. AM62X-SK).
+The system will start, with the console on UART. You can log-in as `root`
+with no password.
+
+# Going further
+
+* [PROCESSOR-SDK-AM62X](https://www.ti.com/tool/PROCESSOR-SDK-AM62X#overview)
+
+# References
+
+* [Buildroot](https://buildroot.org/)
+* [Buildroot reference manual](https://buildroot.org/downloads/manual/manual.html)
+* [Buildroot system development training course](https://bootlin.com/training/buildroot/)
+* [External Buildroot tree ST](https://github.com/bootlin/buildroot-external-st)
```

Comment:
What have we referred from STM??
Ideally we should only refer Buildroot official steps